examples/main/constants/toc.js

```diff
@@ -0,0 +1,83 @@
+// Modules
+import MainExample from '../views/main';
+import NotInteractiveExample from '../views/not-interactive';
+import CustomOverlayExample from '../views/custom-overlay';
+import RouteOverlayExample from '../views/route-overlay';
+import StyleDiffingExample from '../views/style-diffing';
+import ClickExample from '../views/click';
+// Standalone
+import Controls from '../../controls/src/app';
+import GeoJson from '../../geojson/src/app';
+import Interaction from '../../interaction/src/app';
+import Layers from '../../layers/src/app';
+
+export const BASIC_EXAMPLES = 'basicExamples';
+export const STANDALONE_EXAMPLES = 'standalonExamples';
+
+export const DEFAULT_EXAMPLE = 'mainExample';
+
+// TOC
+export default [
```

Comment:
This can be kept here, and what we choose to render as a local example can be configured.